packages/jest-circus/src/legacy-code-todo-rewrite/jestAdapterInit.ts

```diff
@@ -15,7 +16,7 @@ import {
   createEmptyTestResult,
 } from '@jest/test-result';
 import type {Circus, Config, Global} from '@jest/types';
-import {Expect, expect} from 'expect';
+import {expect} from 'expect';
```

Comment:
Better if this would be pulled from `@jest/expect` too.

Reply:
yeah, ideally `jest-circus` and `jest-jasmine2` would both only depend on `@jest/expect` rather than `expect` directly